README.md

```diff
@@ -27,8 +27,7 @@ Change Log
         * Prints the commands submitted to the cluster to `STDOUT` when launched.
     * New `--runfile_prefix` option to help avoid potential filename clashes.
     * Added compatability with GRIDEngine `~/.sge_request files` (by ignoring them). Thanks to [@s-andrews](https://github.com/s-andrews)
-    * New tophat module called `tophat` which introduces a workaround for buggy MAPQ reporting by tophat whilst keeping unique alignments. Thanks to [@FelixKrueger](https://github.com/FelixKrueger).
-        * The previous tophat module is still available if you're not interested in MAPQ scores and would like slightly faster processing. This is now called `tophat_broken_MAPQ.cfmod`.
+    * New tophat module called `tophat_fixedmapq` which introduces a workaround for buggy MAPQ reporting by tophat whilst keeping unique alignments. Thanks to [@FelixKrueger](https://github.com/FelixKrueger).
```

Comment:
Not sure I understand the problem here. We have the tophat module (the fixedmapq one), and the broken tophat_broken_MAPQ.cfmod one (which no one has ever used since), and we don’t have any plans to change this.

From: Phil Ewels [mailto:notifications@github.com]
Sent: 19 January 2015 13:32
To: ewels/clusterflow
Subject: Re: [clusterflow] Merging Simon's BI changes back to 0.4 development branch (#27)


In README.md<https://github.com/ewels/clusterflow/pull/27#discussion-diff-23159736>:

> @@ -27,8 +27,7 @@ Change Log

>          * Prints the commands submitted to the cluster to `STDOUT` when launched.

>      * New `--runfile_prefix` option to help avoid potential filename clashes.

>      * Added compatability with GRIDEngine `~/.sge_request files` (by ignoring them). Thanks to [@s-andrews](https://github.com/s-andrews)

> -    * New tophat module called `tophat` which introduces a workaround for buggy MAPQ reporting by tophat whilst keeping unique alignments. Thanks to [@FelixKrueger](https://github.com/FelixKrueger).

> -        * The previous tophat module is still available if you're not interested in MAPQ scores and would like slightly faster processing. This is now called `tophat_broken_MAPQ.cfmod`.

> +    * New tophat module called `tophat_fixedmapq` which introduces a workaround for buggy MAPQ reporting by tophat whilst keeping unique alignments. Thanks to [@FelixKrueger](https://github.com/FelixKrueger).

I can't see this module? Your fork seems to<https://github.com/s-andrews/clusterflow/tree/master/modules> have tophat.cfmod and tophat_broken_MAPQ.cfmod, the same as mine.

Are you suggesting that we should rename tophat_broken_MAPQ.cfmod back to tophat.cfmod and rename the new version to tophat_fixedmapq.cfmod?

—
Reply to this email directly or view it on GitHub<https://github.com/ewels/clusterflow/pull/27/files#r23159736>.
The Babraham Institute, Babraham Research Campus, Cambridge CB22 3AT Registered Charity No. 1053902.
The information transmitted in this email is directed only to the addressee. If you received this in error, please contact the sender and delete this email from your system. The contents of this e-mail are the views of the sender and do not necessarily represent the views of the Babraham Institute. Full conditions at: www.babraham.ac.uk<http://www.babraham.ac.uk/terms>